Core1Platform/AdministratingCore1/05_EngineRoom.md

```diff
@@ -0,0 +1,239 @@
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened with the current workspace overlapped. It contains four dashlets.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button again to close it.
+   The workspace is visible again.
+
+
+
+## Clear tenant cache
+
+Clearing the tenant cache removes all entries that are currently in the cache of the instance. It is used if there is a general issue in processing, and if changes are made directly through the database rather than through system logic. It is recommended to use this function only if it is really necessary. Clearing the cache will slow down the instance considerably. 
+
+
+#### Prerequisites
+
+- You have administrator rights for the current instance.
+
+#### Procedure
+  
+*Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the [CLEAR TENANT CACHE] button top right.   
+   The following warning message is displayed:   
+
+   ![Warning message](../../Assets/Screenshots/Core1Platform/AdministratingCore1/ClearCacheWarning.png "[Warning message]")
+
+3. Click the [ABBRECHEN] (Cancel) button if you want to cancel the action. 
+
+4. If desired, click the [FORTFAHREN] (Continue) button to clear the cache.   
+   The cache is cleared. Afterwards, it will be rebuilt again. The processes on the instance are significantly slowed down.
+
+
+
+## Check warnings and notices
+
+The platform console provides instance-relevant notices and warning. In opposite to the notifications (see [Check notifications](../UsingCore1/03_GeneralUIFunctions.md#check-notifications "Check notifications")), these notices and warnings are targeted for developers.
+
+
+#### Procedure
+
+*Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the [PLATTFORM-KONSOLE] (Platform console) button.   
+   The *Platform Console* window is opened.
+
+   ![Platform console](../../Assets/Screenshots/Core1Platform/AdministratingCore1/EngineroomPlatformConsole.png "[Platform console]")
+
```

Comment:
Die Balken zum Anomymisieren sind nicht gleich dick. Man kann einen erstellen und dann kopieren und woanders einfügen, so sieht der Screenshots aufgeräumter aus.

Zu 3. Check the messages --> Ist dies wirklich nötig? (Gerade für Entwickler scheint mir der Schritt auch eher überflüssig).

Reply:
das ist eigentlicj der Grund, warum ich das aufrufe